_posts/elements/2015-03-25-example-headings-new.md

```diff
@@ -0,0 +1,50 @@
+---
+layout:       element
+title:        Example - Demo Headings
+date:         2015-02-25 00:00:00
+
+categories:   elements
+tags:
+- example
```

Comment:
Please remove the example tag (it was made for the placeholder posts I added when getting the side rendering up and running).